public/css/theme.css

```diff
@@ -568,8 +563,7 @@ pre.numberSource > code.sourceCode > span {
   counter-increment: source-line;
   padding-left: var(--line-numbers-width);
   text-indent: var(--line-numbers-negative-width);
-  position: relative;
-  left: var(--line-numbers-negative-width);
+  margin-left: var(--line-numbers-negative-width);
```

Comment:
`margin-left` instead of `left` makes it so that the floating newline isn't pushed out by `--line-numbers-negative-width` on lines that aren't highlighted.